<?xml version="1.0"?>
<!-- Note: example launch file. Do not modify! -->
<launch>
  <arg name="loop_rate" default="100"/>

  <node pkg="udom_grasping" type="reactive_grasp_gui"
        name="reactive_grasp_gui_node" output="screen">
    <!-- Node cycle rate (in Hz). -->
    <param name="loop_rate" type="double" value="$(arg loop_rate)"/>
  </node>
</launch>
